docs/usage.md

```diff
@@ -454,6 +454,20 @@ This can also remove 'nonsense' bins of e.g. a single or a collection of very sh
 
 Note that in this context, it is recommended to also set `--min_length_unbinned_contigs` to a suitably high value that corresponds to a reasonable bin size if the `-bin_*_length` parameters are used, so you have useful 'singular' contigs in the unbinned output.
 
+## A note on assembly quality control with ALE
```

Comment:
I think that is nice to have this section, but I feel that it's too verbose in its current form.

I you look a that other "A note on..." sections, you will see that they focus more on the pipeline behavior at a macro level and provide guidance on what to do when things go wrong, instead of going in-depth about a specific tool.

Can you make this more minimal? For example, output is already covered in`docs/output.md`, and you can focus here more on why users should care about ALE results.